src/app/themes/volumio/components/track-manager/volumio-track-manager.html

```diff
@@ -39,9 +39,21 @@
         <div class="smallText">
           {{trackManager.playerService.songLength}}
         </div>
-        <div class="footerText">
-          <span ng-if="trackManager.playerService.state.trackType !== 'webradio'">
-            {{trackManager.playerService.state.trackType}}
+        <div class="footerText" title="Audio Format">
+          <span ng-if="trackManager.playerService.state.trackType !== 'webradio'" ng-switch on="trackManager.playerService.state.trackType">
```

Comment:
Ok, so that's not as straight forward as I thought it might be. Its the kind of thing that would take me all morning, and take you 5 minutes.

If I remove the mark-up and PR just the icons is it something that you could finish with a bit more skill than me?